dag-store.js

```diff
@@ -4,13 +4,10 @@ import uuid from 'node-uuid';
 let nodes = (state = [], action = {}) => {
   switch(action.type) {
```

Comment:
@hocragekey I see. Ok I think I know what you are talking about and I was exactly in the same position while implementing this (as a component in my work project). The state of DAG is a composition of list of nodes and connections. The DAG component is rendered based on the nodes and connections in the DAG. So as a whole when we say the state of the DAG its nothing but nodes & connections and anything beyond that should be maintained outside of it.

But if you insist that you need to have custom properties on each node can you create a specific property under each node, say `customProperties` or `__data__`, and add any property you want? I understand its a hassle maintaining two stores and syncing between each other but at least if someone is debugging or trying figure out what is the state we should not cluter the node state with custom business logic outside the scope of it.